packages/next/lib/typescript/writeAppTypeDeclarations.ts

```diff
@@ -30,8 +30,6 @@ export async function writeAppTypeDeclarations(
   const content =
     '/// <reference types="next" />' +
     eol +
-    '/// <reference types="next/types/global" />' +
-    eol +
     (imageImportsEnabled
       ? '/// <reference types="next/image-types/global" />' + eol
```

Comment:
What about `next/image-types/global`. Can we make that dynamic in the `index.d.ts` file?